code/socialDistribution/views.py

```diff
@@ -421,6 +422,25 @@ def author(request, author_id):
 
     return render(request, 'author/detail.html', context)
 
+def unlisted_posts(request, author_id):
+    """ Display an author's unlisted posts 
+    """
+
+    curr_user = LocalAuthor.objects.get(user=request.user)
+    author = get_object_or_404(LocalAuthor, pk=author_id)
+
+    # TODO: Should become an API request (same as /author/<author-id>) since won't know if author is local/remote
+
+    posts = author.posts.unlisted()
```

Comment:
Since the posts are unlisted, they probably shouldn't be available to other users, meaning we don't need authod_id as a URL parameter. Only want to get the posts for the curr_user I think.